src/ogd/games/THERMOLAB/features/__init__.py

```diff
@@ -9,7 +9,10 @@
     "TaskCompleteCount",
     "ToolNudgeCount",
     "ToolSliderTime",
-
+    "SectionCompleteCount",
+    "TotalPlayTime",
+    "AnswerAttemptsCount",
```

Comment:
We don't have an AnswerAttemptsCount feature here, so we don't want it in the init file as an import.